scripts/make-packages.sh

```diff
@@ -31,8 +31,16 @@ if [ "$OS" == 'Linux' ]; then
         ARCH='x64'
         LIBDIR="lib64"
     else
-        ARCH=x86
         LIBDIR="lib"
```

Comment:
`LIBDIR="lib"` is correct for all of these?

Reply:
it seems like - at least on the images I look at e.g. Ubuntu.

Reply:
hmm. we may do `lib/aarch64-linux-gnu'. let me explore.

Reply:
updated platform fix-up